docs/source/configure.rst

```diff
@@ -4,23 +4,35 @@
 Configuration Overview
 ======================
 
-Gunicorn pulls configuration information from three distinct places.
+Gunicorn pulls configuration information from five distinct places.
 
-The first place that Gunicorn will read configuration from is the framework
+Gunicorn will first read environment variables for the default value of some
+configuration :ref:`settings <settings>`.
+
+The second place that Gunicorn will read configuration from is the framework
 specific configuration file. Currently this only affects Paster applications.
 
-The second source of configuration information is a configuration file that is
-optionally specified on the command line. Anything specified in the Gunicorn
-config file will override any framework specific settings.
+The third source of configuration information is a configuration file that is
+optionally specified using command line arguments. Anything specified in the
+Gunicorn config file will override any framework specific settings.
```

Comment:
Default is `gunicorn.conf.py`. If it's present, it is used implicitly. Maybe we can make that clear here.

Reply:
That's a good idea, I added documentation for this.